ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/tls/TlsConfigurationException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.api.tls;
+
+public class TlsConfigurationException extends RuntimeException {
+  public TlsConfigurationException(final String message) {
```

Comment:
do you want to enable all constructor types? I.e. the ones which take a "cause" exception?

Reply:
Probably not, this is inline with JsonRpcException which is raised if a port is already binded and takes a custom message only (without stacktrace/cause). The primary purpose of this custom exception is to handle invalid keystore/password VertxException raise by .listen() method.

Reply:
refactored as requirements changes to provide overloaded constructor with cause.